platform/platform_tests/distance_tests.cpp

```diff
@@ -28,6 +27,19 @@ struct ScopedSettings
   measurement_utils::Units m_oldUnits;
 };
 
+UNIT_TEST(Distance_InititalDistance)
+{
+  Distance dist;
+  // Run two times to verify that nothing breaks after second format
+  for (Distance const & d : {dist, dist.GetFormattedDistance()})
+  {
+    TEST_EQUAL(d.IsValid(), false, ());
+    TEST_ALMOST_EQUAL_ULPS(d.GetDistance(), -1.0, ());
```

Comment:
Can't say for sure, but probably should put an assert(IsValid()) inside GetDistance ?

Reply:
Not a good idea